### DISCUSSION TITLE
Cross-PLN roadmapping tool demo - Starmaps

### DISCUSSION THREAD

**Discussion by bdehaynin** [memser-spaceport/pl-network-hub#13] [Show and tell]
Starmaps is a cross-project high-level roadmapping tool that reflects Github issues. 
I made a Loom of us at Fission trying it out for the first time, adding in our roadmap for WNFS: https://www.loom.com/share/ff446968fa684ed4b83e98c22ec306d3

**Why is this important?**
Using this tool, we can collaborate more easily with each other and communicate our goals to everyone in the network. It allows to highlight dependencies and plan for what we need ahead of time. 
With each team keeping its projects updated in Starmaps, the network can be made aware of the latest changes easily. It's also a good way to answer questions related to time frame on a project by giving a clear deadline, a question that often comes back. 

**What's the idea?**
The ultimate goal would be to have a single Starmaps tree comprised of all PLN projects linked to each other based on their relationship, but the first step is to have _you_ start using it! 
Let me know if you're interested to use it as well! I'd be happy to help you set it up and answer questions you may have.

**Comment by pandeyanujk** [comment]
@bdehaynin I absolutely second this effort!
